src/convertSchema.test.ts

```diff
@@ -525,6 +527,108 @@ describe('convertSchema', () => {
       },
     })
   })
+  test('Should apply multiple updates in sequence', () => {
```

Comment:
This test doesn't sufficiently test that the output from the first match is passed as the input to the next match. You might want to consider appending to `copy_to` in both matches since this is our internal use case.

Reply:
It does test that -- the `zip` field is changed from string -> number (i.e. long) in the first override, and then the next override changes number -> double for all fields, including `zip`.

Reply:
The test doesn't demonstrate that the previous change is fed into the next matching override. In all cases, you're just setting `bsonType` which shows that all matching overrides are being applied but not that the result of one match is fed into the next match.

Reply:
I think there may be some confusion here, because I think this test case does demonstrate that 🤔 

* The `zip` field is initially `bsonType: 'string'`
* The first override (`*.zip`) matches that field and changes it to `bsonType: 'number'`
* The second override (`*`) _also_ matches that field and conditionally changes the `bsonType` to `double` if it was `number`. (Which this field is, because we passed in the result of the first override.)
* The test case asserts that the `zip` field ends up being `bsonType: 'double'`.

The test case would fail if the chaining behavior didn't work.

Reply:
I'll admit that my example here is contrived. Your `copy_to` example would probably make a better test case, as it's realistic to how we're actually going to use this. It seems like it could be less confusing, as well. I'm happy to change it, if you want.

Reply:
Done: https://github.com/smartprocure/mongo2elastic/pull/20/commits/61226604f91ae8f0f8f6f6e5c720937ee6a424c3

Let me know what you think.